Presentation.Web/app/components/data-processing/data-processing-registration-overview.controller.ts

```diff
@@ -86,6 +86,9 @@
                         });
                         return response;
                     })
+                    .withSchemaFields({
+                        AgreementConcludedAt: { type: "date" }
```

Comment:
Sådan bruges den til at beskrive hvilken type en property er. Hvilket skal bruges for at kunne få de rigtige filtrerings operatorer